cypress/plugins/deployProject.js

```diff
@@ -36,9 +37,9 @@ const deployOnNetlify = ({ project }, config) => {
   process.stdout.write(`Deploying project: ${project}...`);
 
   // Trigger deploy
-  const deploy = spawnSync("npm", ["run", "deploy"], {
+  const deploy = execa.sync("npm", ["run", "deploy"], {
     cwd: join(config.buildsFolder, project),
-    encoding: "utf-8",
```

Comment:
Not needed.
https://github.com/sindresorhus/execa/blob/2706b3f878345d99e890ff3261d9a683ebcd7dfe/index.js#L41